grammar.js

```diff
@@ -120,14 +120,18 @@ module.exports = grammar({
     ),
 
     let_expr: $ => seq(
+      field("var", $.let_in_block),
```

Comment:
Nitpick: "var" is a bit off to talk about a binding `(var, term)` (plus other potential annotations).